collectors/ebpf.plugin/ebpf.c

```diff
@@ -437,7 +442,7 @@ void ebpf_create_charts_on_apps(char *id, char *title, char *units, char *family
                                 char *algorithm, struct target *root)
 {
     struct target *w;
-    ebpf_write_chart_cmd(NETDATA_APPS_FAMILY, id, title, units, family, "stacked", order);
+    ebpf_write_chart_cmd(NETDATA_APPS_FAMILY, id, title, units, family, "stacked", "''", order);
```

Comment:
I would find more natural to be able to set this parameter to NULL and then function `ebpf_write_chart_cmd` to output `''` when it sees that parameter is NULL.

Reply:
I agree 100% with you, but I thought it would be more readable.
After you wrote this, I am more comfortable to  use the NULL.